ircbot/plugins/translate.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import json
+
+from langdetect import detect, detect_langs
+from ircbot import bot
+
+
+def translate(bot, text):
+    api_key = bot.config['Yandex']['translate_key']
+    lang = detect(text)
+    translate_url = "https://translate.yandex.net/api/v1.5/tr.json/translate?key={}&text={}&lang={}-{}".format(api_key, text, lang, bot.config["System"]["lang"])
+    api_response = requests.get(translate_url).text
+    api_json = json.loads(api_response)
+    print(api_json)
```

Comment:
Why do you print this?

Reply:
this is an edited version of the previous function solely for translating russian, in the original it had been printed so i left it